flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/scala/batch/table/AggregationsITCase.scala

```diff
@@ -342,4 +342,85 @@ class AggregationsITCase(
     TestBaseUtils.compareResultAsText(results.asJava, expected)
   }
 
+  @Test(expected = classOf[UnsupportedOperationException])
```

Comment:
I created a new package for runtime IT cases. You could move the new tests here in `org.apache.flink.table.runtime.dataset.DataSetWindowAggregateITCase`.